src/lib.rs

```diff
@@ -325,14 +325,44 @@ where
     }
 }
 
+impl<K, V, S: BuildHasher> FlurryHashMap<K, V, S> {
+    /// Creates a new, empty map with the default initial table size (16).
+    /// It uses the given hasher, rather than the default `RandomState`.
+    pub fn new_with_hasher(hasher: S) -> Self {
```

Comment:
This method is traditionally called [`with_hasher`](https://doc.rust-lang.org/std/collections/struct.HashMap.html#method.with_hasher). It'd be good to match the documentation to that of the standard library for the same method!